src/Components/HomePage/ProductElement.js

```diff
@@ -1,31 +1,38 @@
-import React from 'react';
-import { useMyCartContext } from "Context/GlobalContext";
-import Button from 'Components/Button/Button';
-import { Link } from 'react-router-dom';
-import { ProductHoverButton, Article, Text, Name, SKU, ImageContainer } from './styled';
 
-const ProductElement = ({id, data, showCategory, showShortDescription}) => {
+// This feels weird
+/* eslint-disable camelcase */
+import React from 'react'
+import PropTypes from 'prop-types'
+import { useMyCartContext } from 'Context/GlobalContext'
+import Button from 'Components/Button/Button'
+import { Link } from 'react-router-dom'
+import { ProductHoverButton, Article, Text, Name, SKU, ImageContainer } from './styled'
 
-    const { addToMyCart } = useMyCartContext();
+const ProductElement = ({ id, data, showCategory, showShortDescription }) => {
+    const { addToMyCart } = useMyCartContext()
 
-    const productId = id;
+    const productId = id
 
     const {
         name,
         sku,
         category,
         mainimage,
         price,
+        // Again, not sure if this is ok
+        // eslint-disable-next-line camelcase
```

Comment:
It is ok, some APIs save their properties with snake_case, and eslint doesn't like snake_casing. So for this particular use case it is valid to disable the rule.